lib/DBM_Filter.pm

```diff
@@ -4,7 +4,8 @@ use strict;
 use warnings;
 our $VERSION = '0.06';
 
-package Tie::Hash ;
+package
+    Tie::Hash ;
 
 use strict;
 use warnings;
```

Comment:
These are redundant and could be deleted.

Reply:
done